forte/processors/data_augment/base_op_processor.py

```diff
@@ -0,0 +1,299 @@
+                )
+            )
+
+        new_packs: List[Tuple[str, DataPack]] = []
+
+        for aug_pack_name in aug_pack_names:
+            new_pack_name: str = self.configs["augment_pack_names"].get(
+                aug_pack_name, "augmented_" + aug_pack_name
+            )
+            data_pack = input_pack.get_pack(aug_pack_name)
+
+            new_packs.append(
+                (new_pack_name, self._new_data_packs[data_pack.pack_id])
+            )
+
+        for new_pack_name, new_pack in new_packs:
+            input_pack.add_pack_(new_pack, new_pack_name)
+
+        # Copy the MultiPackLinks/MultiPackGroups
+        for mpl in input_pack.get(MultiPackLink):
+            self._copy_multi_pack_link_or_group(mpl, input_pack)
+        for mpg in input_pack.get(MultiPackGroup):
+            self._copy_multi_pack_link_or_group(mpg, input_pack)
+
+        # Must be called after processing each multipack
+        # to reset internal states.
+        self._clear_states()
+
+    @classmethod
+    def default_configs(cls):
+        """
+        Returns:
+            A dictionary with the default config for this processor.
+        Following are the keys for this dictionary:
+            - type:
+                Should not modify this field, in order to use the kwargs.
+            - data_aug_op:
+                The data augmentation Op for the processor.
+                It should be a full qualified class name.
+                Example:
+                    "forte.processors.data_augment.algorithms.
+                    text_replacement_op.TextReplacementOp"
+            - data_aug_op_config:
+                The configuration for data augmentation Op.
+                Example:
+                    .. code-block:: python
+                        'data_aug_op_config': {
+                            'lang': 'en',
+                            'use_gpu': False,
+                            'other_entry_policy': {
+                                'ft.onto.base_ontology.Document': 'auto_align',
+                                'ft.onto.base_ontology.Sentence': 'auto_align',
+                            }
+                        }
+            - augment_pack_names:
+                A dict specifies the DataPacks to augment and their output
+                names. It should be key-value pairs where the key is the
+                input DataPack name, and the value is the output DataPack
+                name after augmentation.
+                If empty, all the DataPacks will be augmented, and the output
+                names will be automatically generated by prepending
+                an `'augmented_'` prefix.
+                Example:
+                    .. code-block:: python
+                        'data_aug_op_config': {
+                            'src': 'aug_src',
+                            'tgt': 'aug_tgt'
+                        }
+        """
+        return {
+            "type": "data_augmentation_op",
```

Comment:
I think we have gone through this before. Since we don't need to use `kwargs` anymore, we shouldn't need this `type`?